teszt.md

```diff
@@ -1,16 +1,42 @@
 # Teszt
 
-Név: 
+Név: Dumitras István
 
 ## Kérdések:
 
 1. Mit értünk egy `commit` alatt, mit tartalmaz?
+commit: a commitban adjuk meg a commentet amiben változtatásokat írunk le, mik módosultak az előző kódhoz képest. Cégtől függően adhatunk neki projekt számot is pl #107_header_modified. Célszerű minél részletesebben leírni mit módosítottunk, hogy abban az esetben ha egy másik programozó veszi át akkor könynebben lássa át. megadhatunk git commit -m "" ekkor csak a címet írjuk meg de enterrel szintén adhatunk több sort is, vagy git commit ahol viszont az elso sor a cím és a többibe pedig a leirast irhatjuk meg.
+
 1. Mi a különbség a `fetch` es `pull` git parancsok között?
+git fetch: a fetch nem mergel bele a local repoba ezt kézzel kell megtenni.
+git pull: a pull az fetchel is. Maga a locális repoba mergeli a távoli repot.
+
 1. Mi a három állapota file-oknak git szempontjából, milyen parancsokkal mozgatjuk ezek között?
+1.1 git add . (vagy filenév.kiterjesztes)--> feltesszük a stage-re
+	amugy git resettel levehetjük.
+1.2 git commit -m vagy simán git commit itt leírást készítünk, hogy mi milyen változasok történtek.
+1.3 git push:  feltoljuk a távoli repoba (és reménykedünk hogy nem lesz conflict (:  )
```

Comment:
0.5✓
Elso sorban itt az untracked, modified es staged allapotokra lettem volna kivancsi, de az 1. pontban erintetted ezeket is ugyhogy jar pont.
Megjegyzes: push elott szoktuk kihuzni a tavoli valtoztatasokat es oldjuk fel a conflictot, ha van conflict akkor a szerver siman elutasitja a push-t.